### PR TITLE
Rules

### PR DESCRIPTION
**Pull request by aishwarya-bm**
1. rules pages created
2. added share option on category card in home page
3. minor CSS styles added

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
All the changes look good !! Good work Aishwarya!! You can deploy it on Netlify so that we can the get to see the live preview

File: screens/home.html [hunk 1]
```diff
@@ -36,23 +36,23 @@ <h2 class="text-left"><i>Quiz app</i></h2>
             <h4 class="page-heading text-center">Choose a category</h4>
```
Comment (vaibhav-mahalle) [inline screens/home.html:36]: it would be helpful to review if preview is added.

**Review by vaibhav-mahalle — APPROVED** [review]
everything looks good. please add preview for the reference in future pull requests.

File: screens/home.html (hunk 1 above)
Comment (aishwarya-bm) [reply screens/home.html:36]: I forgot to add the screenshots which I took, will remember next time

**Comment by aishwarya-bm** [comment]
> 
 I forgot to add the screenshots which I took, will remember next time

**Pull request merged**